3_javascript/javascript1/week1/javascript.js

```diff
@@ -0,0 +1,120 @@
+console.log('------------- Age - ify(A future age calculator)-------------');
+
+let yearOfBirth = 1976;
+let yearFuture = 2076;
+let age = yearFuture - yearOfBirth;
+
+console.log('You will be ' + age + ' years old in ' + yearFuture);
```

Comment:
Here I will suggest to use template literals, they make it easier to interpolate stringsmore elegantly . You can read about it [here] :)(https://www.w3schools.com/js/js_string_templates.asp).
```suggestion
console.log(`You will be ${age} years old in ${yearFuture}`);
```

Reply:
Hi @margarit,
Thank you so much :). I have made it correct.